package.json

```diff
@@ -58,6 +58,12 @@
     "redux-react-firebase"
   ],
   "dependencies": {
+    "gitbook-plugin-anchorjs": "^2.1.0",
```

Comment:
We can remove all of the gitbook dependencies - for some reason their cli automatically installs them